source/documentation/upgrade_guide/topics/Installing_Virtual_Product_Hypervisors_on_local_RHEL.adoc

```diff
@@ -67,7 +67,7 @@ sslclientcert = /etc/pki/entitlement/5659494963772844103.pem
 sslclientkey = /etc/pki/entitlement/5659494963772844103-key.pem
 ----
 
-. Start your reposync in an appropriate directory:
+. Start your `reposync` in an appropriate directory:
```

Comment:
I don't remember which style guide mentions it, but we shouldn't use commands or GUI items as words within sentences, other than to refer to them as commands or GUI items. So in this case:
```suggestion
. Run the `reposync` command in an appropriate directory:
```
or

```suggestion
. Start synchronizing your repository in an appropriate directory:
```